java/org/prism/ParsingOptions.java

```diff
@@ -0,0 +1,82 @@
+package org.prism;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+// @formatter:off
+public class ParsingOptions {
```

Comment:
There is a check in TruffleRuby that every class that can be marked as `final` is marked as such.
Could you make this class and Parser too `abstract` since they are not meant to be instantiated (at least currently)?
That will satisfy that check.

Reply:
Correction, the check currently only looks at `src/main` and so not `src/yarp`.
But it seems good to mark as `abstract` for clarity anyway.

Reply:
Done